Open62541.xs

```diff
@@ -269,6 +269,39 @@ server_run_mgset(pTHX_ SV* sv, MAGIC* mg)
 
 static MGVTBL server_run_mgvtbl = { 0, server_run_mgset, 0, 0, 0, 0, 0, 0 };
 
+/* Open62541 C callback handling */
+
+typedef struct {
+	SV *			pcc_callback;
+	SV *			pcc_client;
+	SV *			pcc_data;
+}				PerlClientCallback;
+
+static void
+UA_ClientAsyncServiceCallbackPerl(UA_Client *client, void *userdata,
```

Comment:
The UA_ prefix is the namespace of the open62541 library.  We should not name our functions in their namespace.  As the function is static, we do not need a namespace anyway.  Just omit the UA_.